generics-eot.cabal

```diff
@@ -54,6 +54,7 @@ test-suite quickcheck
     , directory
     , filepath
     , QuickCheck
+    , void
```

Comment:
This library uses `hpack` to generate the cabal file. Could you please make your changes there and generate the cabal file?

Reply:
This will (hopefully) also fix the CI failures.